pkgs/tools/backup/pgbackrest/default.nix

```diff
@@ -0,0 +1,40 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  pkg-config,
+  postgresql,
+  openssl,
+  lz4,
+  bzip2,
+  libxml2,
+  zlib,
+  zstd,
+  libyaml,
+}:
+stdenv.mkDerivation rec {
+  pname = "pgbackrest";
+  version = "2.40";
+
+  src = fetchFromGitHub {
+    owner = "pgbackrest";
+    repo = "pgbackrest";
```

Comment:
```suggestion
    repo = pname;
```

Reply:
I think I had multiple discussions in the past about this, and especially @SuperSandro2000 was against this kind of substituting everything with variables. Is there some sort of standard nowadays about this?

Reply:
> Is there some sort of standard nowadays about this?

I have no idea. Currently, there are 4069 packages with `repo = pname` but maybe it's not a big deal to leave it like you did.